### PR TITLE
LPS-155969 Tables with ListType foreign key should use listTypeId as its column name

### PR DESCRIPTION
**Pull request by gislayne-vitorino**

### CODE REVIEW

**Comment by gislayne-vitorino** [comment]
ci:test:relevant
ci:test:sf

**Review by drewbrokke — APPROVED** [review]

**Comment by gislayne-vitorino** [comment]
ci:test:sf
ci:test:relevant

**Comment by gislayne-vitorino** [comment]
ci:test:sf
ci:test:relevant

**Comment by gislayne-vitorino** [comment]
ci:test:relevant

**Comment by drewbrokke** [comment]
ci:test:relevant

**Comment by epark18** [comment]
It looks like most of these tests are failing with this warning:
`LIFERAY_ERROR: No JSON web service action with path /user/add-user and method POST for null` 
This error is thrown in almost every test that is using the _JSONUser.addUser_ macro. 

One example would be the test [Locale#ViewUserPreferredLocale](https://testray.liferay.com/home/-/testray/case_results/1903584879). Going off of the [Poshi Report](https://testray.liferay.com/reports/production/logs/2022-07/test-1-38/test-portal-acceptance-pullrequest(master)/2216/functional-tomcat90-mysql57-jdk8/21/6/LocalFile.Locale_ViewUserPreferredLocale/index.html.gz?authuser=0), it looks like a user is never created after using the JSON macro. Here is a snippet of the error.
 ```     
[exec]     Executing commands: curl -X POST -u 'test@liferay.com:test' -d 'companyId=46662' -d 'autoPassword=******** -d 'password1=******** -d 'password2=******** -d 'autoScreenName=false' -d 'screenName=usersn' -d 'emailAddress=userea@liferay.com' -d 'facebookId=0' -d 'openId=' -d 'locale=' -d 'firstName=userfn' -d 'middleName=' -d 'lastName=userln' -d 'prefixId=0' -d 'suffixId=0' -d 'male=true' -d 'birthdayMonth=4' -d 'birthdayDay=31' -d 'birthdayYear=1986' -d 'jobTitle=' -d 'groupIds=' -d 'organizationIds=' -d 'roleIds=' -d 'userGroupIds=' -d 'sendEmail=true' -d 'serviceContext={"addGroupPermissions" : true, "addGuestPermissions": true}' http://test-1-33-40.lax.liferay.com:8080/api/jsonws/user/add-user
[beanshell] 2022-07-11 23:15:03.855 ERROR [http-nio-8080-exec-4][JSONWebServiceServiceAction:126] No JSON web service action with path /user/add-user and method POST for null
     [exec]     {}Finished executing commands.
``` 
[Logs](https://testray.liferay.com/reports/production/logs/2022-07/test-1-38/test-portal-acceptance-pullrequest(master)/2216/functional-tomcat90-mysql57-jdk8/21/6/jenkins-console.txt.gz?authuser=0)

Same thing is happening in [CPUsergroups#RemoveMember](https://testray.liferay.com/home/-/testray/case_results/1903580019) but it has this error in the [logs ](https://testray.liferay.com/reports/production/logs/2022-07/test-1-38/test-portal-acceptance-pullrequest(master)/2216/functional-tomcat90-mysql57-jdk8/7/18/jenkins-console.txt.gz?authuser=0)
```   
   [exec]     BUILD FAILED: No results for path: $['userId']
     [exec]     /opt/dev/projects/github/liferay-portal/portal-web/test/functional/com/liferay/portalweb/macros/json/user/JSONUserAPI.macro[_getUserIdByEmailAddress]:236
     [exec]     /opt/dev/projects/github/liferay-portal/portal-web/test/functional/com/liferay/portalweb/macros/json/user/JSONUserSetter.macro[setUserId]:94
     [exec]     /opt/dev/projects/github/liferay-portal/portal-web/test/functional/com/liferay/portalweb/macros/json/user/JSONUser.macro[addUserToUserGroup]:158
     [exec]     /opt/dev/projects/github/liferay-portal/portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/cpusergroups/CPUsergroups.testcase[RemoveMember]:194
     [exec] 
     [exec] 
     [exec] com.liferay.poshi.runner.PoshiRunner > test[LocalFile.CPUsergroups#RemoveMember] STANDARD_ERROR
     [exec]     java.lang.Exception: No results for path: $['userId']
     [exec]     	at com.liferay.poshi.core.util.ExternalMethod.execute(ExternalMethod.java:46)
     [exec]     	at com.liferay.poshi.core.util.ExternalMethod.execute(ExternalMethod.java:91)
     [exec]     	at com.liferay.poshi.core.PoshiGetterUtil.getMethodReturnValue(PoshiGetterUtil.java:299)
     [exec]     	at com.liferay.poshi.runner.PoshiRunnerExecutor.getVarMethodValue(PoshiRunnerExecutor.java:1102)
     [exec]     	at com.liferay.poshi.runner.PoshiRunnerExecutor._getVarValue(PoshiRunnerExecutor.java:1226)
     [exec]     	at com.liferay.poshi.runner.PoshiRunnerExecutor.runCommandVarElement(PoshiRunnerExecutor.java:244)
     [exec]     	at com.liferay.poshi.runner.PoshiRunnerExecutor.parseElement(PoshiRunnerExecutor.java:227)
     [exec]     	at com.liferay.poshi.runner.PoshiRunnerExecutor.runMacroCommandElement(PoshiRunnerExecutor.java:681)
     [exec]     	at com.liferay.poshi.runner.PoshiRunnerExecutor.runMacroExecuteElement(PoshiRunnerExecutor.java:715)
     [exec]     	at com.liferay.poshi.runner.PoshiRunnerExecutor.parseElement(PoshiRunnerExecutor.java:196)
     [exec]     	at com.liferay.poshi.runner.PoshiRunnerExecutor.runMacroCommandElement(PoshiRunnerExecutor.java:681)
     [exec]     	at com.liferay.poshi.runner.PoshiRunnerExecutor.runMacroExecuteElement(PoshiRunnerExecutor.java:715)
     [exec]     	at com.liferay.poshi.runner.PoshiRunnerExecutor.parseElement(PoshiRunnerExecutor.java:196)
     [exec]     	at com.liferay.poshi.runner.PoshiRunnerExecutor.runMacroCommandElement(PoshiRunnerExecutor.java:681)
     [exec]     	at com.liferay.poshi.runner.PoshiRunnerExecutor.runMacroExecuteElement(PoshiRunnerExecutor.java:715)
     [exec]     	at com.liferay.poshi.runner.PoshiRunnerExecutor.parseElement(PoshiRunnerExecutor.java:196)
     [exec]     	at com.liferay.poshi.runner.PoshiRunnerExecutor.runTestCaseCommandElement(PoshiRunnerExecutor.java:974)
     [exec]     	at com.liferay.poshi.runner.PoshiRunner._runNamespacedClassCommandName(PoshiRunner.java:314)
     [exec]     	at com.liferay.poshi.runner.PoshiRunner._runCommand(PoshiRunner.java:290)
     [exec]     	at com.liferay.poshi.runner.PoshiRunner.test(PoshiRunner.java:266)
     [exec]     	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
     [exec]     	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
     [exec]     	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
     [exec]     	at java.lang.reflect.Method.invoke(Method.java:498)
     [exec]     	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
     [exec]     	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
     [exec]     	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
     [exec]     	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
     [exec]     	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
     [exec]     	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
     [exec]     	at com.liferay.poshi.runner.PoshiRunner$RetryTestRule$RetryStatement.evaluate(PoshiRunner.java:368)
     [exec]     	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
     [exec]     	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
     [exec]     	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
     [exec]     	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
     [exec]     	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
     [exec]     	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
     [exec]     	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
     [exec]     	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
     [exec]     	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
     [exec]     	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
     [exec]     	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
     [exec]     	at org.junit.runners.Suite.runChild(Suite.java:128)
     [exec]     	at org.junit.runners.Suite.runChild(Suite.java:27)
     [exec]     	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
     [exec]     	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
     [exec]     	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
     [exec]     	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
     [exec]     	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
     [exec]     	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
     [exec]     	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
     [exec]     	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
     [exec]     	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.runTestClass(JUnitTestClassExecutor.java:110)
     [exec]     	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.execute(JUnitTestClassExecutor.java:58)
     [exec]     	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.execute(JUnitTestClassExecutor.java:38)
     [exec]     	at org.gradle.api.internal.tasks.testing.junit.AbstractJUnitTestClassProcessor.processTestClass(AbstractJUnitTestClassProcessor.java:62)
     [exec]     	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:51)
     [exec]     	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
     [exec]     	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
     [exec]     	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
     [exec]     	at java.lang.reflect.Method.invoke(Method.java:498)
     [exec]     	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:36)
     [exec]     	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
     [exec]     	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:33)
     [exec]     	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:94)
     [exec]     	at com.sun.proxy.$Proxy2.processTestClass(Unknown Source)
     [exec]     	at org.gradle.api.internal.tasks.testing.worker.TestWorker.processTestClass(TestWorker.java:119)
     [exec]     	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
     [exec]     	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
     [exec]     	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
     [exec]     	at java.lang.reflect.Method.invoke(Method.java:498)
     [exec]     	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:36)
     [exec]     	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
     [exec]     	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:182)
     [exec]     	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:164)
     [exec]     	at org.gradle.internal.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:414)
     [exec]     	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:64)
     [exec]     	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:48)
     [exec]     	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
     [exec]     	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
     [exec]     	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:56)
     [exec]     	at java.lang.Thread.run(Thread.java:745)
     [exec]     Caused by: java.lang.reflect.InvocationTargetException
     [exec]     	at sun.reflect.GeneratedMethodAccessor11.invoke(Unknown Source)
     [exec]     	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
     [exec]     	at java.lang.reflect.Method.invoke(Method.java:498)
     [exec]     	at com.liferay.poshi.core.util.ExternalMethod.execute(ExternalMethod.java:40)
     [exec]     	... 81 more
     [exec]     Caused by: com.jayway.jsonpath.PathNotFoundException: No results for path: $['userId']
     [exec]     	at com.jayway.jsonpath.internal.path.EvaluationContextImpl.getValue(EvaluationContextImpl.java:131)```

**Comment by gislayne-vitorino** [comment]
ci:test:sf 
ci:test:relevant

**Comment by epark18** [comment]
ci:test:relevant

**Comment by Sharryxue** [comment]
ci:test:relevant

**Comment by epark18** [comment]
The issues look similar to earlier and are likely related to failing JSON macros. Basing it off this functional test [JSONSmoke#RunAllJSONMacros](https://testray.liferay.com/home/-/testray/case_results/1906067939); there are failures when trying to add an organization. [Console Log](https://testray.liferay.com/reports/production/logs/2022-07/test-1-37/test-portal-acceptance-pullrequest(master)/3402/functional-tomcat90-mysql57-jdk8/20/18/jenkins-console.txt.gz?authuser=0) |

```
     [exec]     LIFERAY_ERROR: Conversion failed: ; <--- java.lang.NumberFormatException: For input string: ""
     [exec]     Executing commands: curl -X GET -u 'test@liferay.com:test' http://test-1-36-62.lax.liferay.com:8080/api/jsonws/company/get-company-by-virtual-host/virtual-host/test-1-36-62.lax.liferay.com
     [exec]     {"active":true,"companyId":"50736","createDate":1657766273177,"homeURL":"","industry":"","legalId":"","legalName":"","legalType":"","logoId":"0","maxUsers":0,"modifiedDate":1657766273177,"mvccVersion":"1","mx":"liferay.com","name":"test-1-36-62.lax.liferay.com","sicCode":"","size":"","system":false,"tickerSymbol":"","type":"","userId":"0","userName":"","webId":"test-1-36-62.lax.liferay.com"}Finished executing commands.
     [exec] 
     [exec]     Executing commands: curl -X GET -u 'test@liferay.com:test' http://test-1-36-62.lax.liferay.com:8080/api/jsonws/organization/get-organizations/company-id/50736/parent-organization-id/0
     [exec]     []Finished executing commands.
     [exec] 
     [exec]     Executing commands: curl -X POST -u 'test@liferay.com:test' -d 'organizationId=' http://test-1-36-62.lax.liferay.com:8080/api/jsonws/organization/delete-organization
[beanshell] 2022-07-14 02:40:28.571 ERROR [http-nio-8080-exec-2][JSONWebServiceServiceAction:126] Conversion failed: ; <--- java.lang.NumberFormatException: For input string: ""
     [exec]     {}Finished executing commands.
     [exec] 
     [exec]     LIFERAY_ERROR: Conversion failed: ; <--- java.lang.NumberFormatException: For input string: ""
     [exec]     Executing commands: curl -X GET -u 'test@liferay.com:test' http://test-1-36-62.lax.liferay.com:8080/api/jsonws/company/get-company-by-virtual-host/virtual-host/test-1-36-62.lax.liferay.com
     [exec]     {"active":true,"companyId":"50736","createDate":1657766273177,"homeURL":"","industry":"","legalId":"","legalName":"","legalType":"","logoId":"0","maxUsers":0,"modifiedDate":1657766273177,"mvccVersion":"1","mx":"liferay.com","name":"test-1-36-62.lax.liferay.com","sicCode":"","size":"","system":false,"tickerSymbol":"","type":"","userId":"0","userName":"","webId":"test-1-36-62.lax.liferay.com"}Finished executing commands.
     [exec] 
     [exec]     Executing commands: curl -X GET -u 'test@liferay.com:test' http://test-1-36-62.lax.liferay.com:8080/api/jsonws/organization/get-organizations/company-id/50736/parent-organization-id/0
     [exec]     []Finished executing commands.
     [exec] 
     [exec]     Executing commands: curl -X GET -u 'test@liferay.com:test' http://test-1-36-62.lax.liferay.com:8080/api/jsonws/organization/get-organization/organization-id/
[beanshell] 2022-07-14 02:40:31.220 ERROR [http-nio-8080-exec-9][JSONWebServiceServiceAction:126] Missing value for parameter organization-id
     [exec]     {}Finished executing commands.
     [exec] 
     [exec] 
     [exec]     BUILD FAILED: No results for path: $['name']
     [exec]     /opt/dev/projects/github/liferay-portal/portal-web/test/functional/com/liferay/portalweb/macros/json/organizations/JSONOrganization.macro[getOrganizationName]:116
     [exec]     /opt/dev/projects/github/liferay-portal/portal-web/test/functional/com/liferay/portalweb/macros/json/organizations/JSONOrganizationTest.macro[runSmoke]:32
     [exec]     /opt/dev/projects/github/liferay-portal/portal-web/test/functional/com/liferay/portalweb/tests/enduser/json/JSONSmoke.testcase[RunAllJSONMacros]:41
     [exec] 
     [exec] 
     [exec] com.liferay.poshi.runner.PoshiRunner > test[LocalFile.JSONSmoke#RunAllJSONMacros] STANDARD_ERROR
     [exec]     java.lang.Exception: No results for path: $['name']
     [exec]     	at com.liferay.poshi.core.util.ExternalMethod.execute(ExternalMethod.java:46)
     [exec]     	at com.liferay.poshi.core.util.ExternalMethod.execute(ExternalMethod.java:91)
```

File: modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/upgrade/v8_5_0/CommerceAddressTypeUpgradeProcess.java [hunk 1]
```diff
@@ -60,13 +60,13 @@ private long _getListTypeId(String name) {
...
-	private void _setAddressType(long newTypeId, long oldTypeId)
+	private void _setAddressListType(long newListTypeId, long oldListTypeId)
 		throws Exception {
 
 		runSQL(
 			StringBundler.concat(
-				"update Address set typeId = ", newTypeId, " where typeId = ",
-				oldTypeId));
+				"update Address set listTypeId = ", newListTypeId,
+				" where lisTypeId = ", oldListTypeId));
```
Comment (epark18) [inline modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/upgrade/v8_5_0/CommerceAddressTypeUpgradeProcess.java:69]: Hi @gislayne-vitorino, I think this typo is causing a couple of failures.

**Comment by epark18** [comment]
[SAML#ViewSPInitiatedSSOWithSAMLAttributeUUIDUsedForUserMatchingMultipleSP
](https://testray.liferay.com/home/-/testray/case_results/1908633308)
[SAML#ViewSPInitiatedSSOwithSAMLAttributeUUIDUsedForUserMatching](https://testray.liferay.com/home/-/testray/case_results/1908633188)
look like unique failures. Both are failing with the same error [Error Logs](https://testray.liferay.com/reports/production/logs/2022-07/test-1-38/test-portal-acceptance-pullrequest(master)/2342/functional-tomcat90-mysql57-jdk8/1/8/jenkins-console.txt.gz?authuser=0).

```
     [exec] 2022-07-15 18:58:49.165 DEBUG [http-nio-8080-exec-7][WebSsoProfileImpl:1664] SAML authenticated user user1@liferay.com
     [exec] 2022-07-15 18:58:49.167 DEBUG [http-nio-8080-exec-7][DefaultUserResolver:80] Resolving user with name ID format urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress and value user1@liferay.com
     [exec] 2022-07-15 18:58:49.170 DEBUG [http-nio-8080-exec-7][AttributeUserFieldExpressionResolver:72] Resolving user with user field expression: uuid
     [exec] 2022-07-15 18:58:49.171 DEBUG [http-nio-8080-exec-7][DefaultUserResolver:263] User identifier expression is mapped to SAML attribute value "434823f2-65f0-8ea2-d777-3fb0f32e8d42"
     [exec] 2022-07-15 18:58:49.175 DEBUG [http-nio-8080-exec-7][DefaultUserFieldExpressionHandler:148] null
     [exec] com.liferay.portal.kernel.exception.NoSuchUserException: No User exists with the key {uuid=434823f2-65f0-8ea2-d777-3fb0f32e8d42, companyId=43339}
     [exec] 	at com.liferay.portal.service.persistence.impl.UserPersistenceImpl.findByUuid_C_First(UserPersistenceImpl.java:877) ~[portal-impl.jar:?]
     [exec] 	at com.liferay.portal.service.base.UserLocalServiceBaseImpl.getUserByUuidAndCompanyId(UserLocalServiceBaseImpl.java:472) ~[portal-impl.jar:?]
     [exec] 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_121]
     [exec] 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_121]
     [exec] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_121]
     [exec] 	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_121]
     [exec] 	at com.liferay.portal.kernel.bean.ClassLoaderBeanHandler.invoke(ClassLoaderBeanHandler.java:66) ~[portal-kernel.jar:?]
     [exec] 	at com.sun.proxy.$Proxy1146.getUserByUuidAndCompanyId(Unknown Source) ~[?:?]
     [exec] 	at com.liferay.portal.kernel.service.UserLocalServiceWrapper.getUserByUuidAndCompanyId(UserLocalServiceWrapper.java:2003) ~[portal-kernel.jar:?]
     [exec] 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_121]
     [exec] 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_121]
     [exec] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_121]
     [exec] 	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_121]
     [exec] 	at com.liferay.portal.kernel.bean.ClassLoaderBeanHandler.invoke(ClassLoaderBeanHandler.java:66) ~[portal-kernel.jar:?]

     [exec] 2022-07-15 18:58:49.191 DEBUG [http-nio-8080-exec-7][DefaultUserResolver:108] Adding user with attributes map {firstName=[user], lastName=[1], emailAddress=[user1@liferay.com], screenName=[user1], uuid=[434823f2-65f0-8ea2-d777-3fb0f32e8d42]}
     [exec] 2022-07-15 18:58:49.990 DEBUG [http-nio-8080-exec-7][DefaultUserResolver:138] Added user {"mvccVersion": 3, "ctCollectionId": 0, "uuid": "434823f2-65f0-8ea2-d777-3fb0f32e8d42", "externalReferenceCode": "434823f2-65f0-8ea2-d777-3fb0f32e8d42", "userId": 45750, "companyId": 43339, "createDate": "Fri Jul 15 18:58:49 GMT 2022", "modifiedDate": "Fri Jul 15 18:58:49 GMT 2022", "defaultUser": false, "contactId": 45751, "password": "{PBKDF2WITHHMACSHA1}AAAAoAAK/IDyKD+y4TBGLo+mJJENjA1tYWplCGQAUiXVh+iI", "passwordEncrypted": true, "passwordReset": false, "passwordModifiedDate": "Fri Jul 15 18:58:49 GMT 2022", "digest": "c5a981abac546f273f1a4600885e36ed,0983a76ceaed5efc77a2f65d950660ed,48efa02cb428c8ed827e3fb1ae49ca92", "reminderQueryQuestion": "", "reminderQueryAnswer": "", "graceLoginCount": 0, "screenName": "user1", "emailAddress": "user1@liferay.com", "facebookId": 0, "googleUserId": "", "ldapServerId": -1, "openId": "", "portraitId": 0, "languageId": "en_US", "timeZoneId": "UTC", "greeting": "Welcome user 1!", "comments": "", "firstName": "user", "middleName": "", "lastName": "1", "jobTitle": "", "loginDate": null, "loginIP": "", "lastLoginDate": null, "lastLoginIP": "", "lastFailedLoginDate": null, "failedLoginAttempts": 0, "lockout": false, "lockoutDate": null, "agreedToTermsOfUse": false, "emailAddressVerified": true, "status": 0}
     [exec] 2022-07-15 18:59:05.756 ERROR [http-nio-8080-exec-7][BeanPropertiesImpl:297] null
     [exec] jodd.bean.BeanException: Simple property not found: prefixId. Invalid property: ContactImpl#prefixId (ContactImpl#prefixId)
     [exec] 	at jodd.bean.BeanUtilBean.getSimpleProperty(BeanUtilBean.java:206) ~[jodd-util.jar:6.0.1]
     [exec] 	at jodd.bean.BeanUtilBean._getIndexProperty(BeanUtilBean.java:307) ~[jodd-util.jar:6.0.1]
     [exec] 	at jodd.bean.BeanUtilBean.getIndexProperty(BeanUtilBean.java:299) ~[jodd-util.jar:6.0.1]
     [exec] 	at jodd.bean.BeanUtilBean.getProperty(BeanUtilBean.java:511) ~[jodd-util.jar:6.0.1]
     [exec] 	at com.liferay.portal.bean.BeanPropertiesImpl.getInteger(BeanPropertiesImpl.java:292) ~[portal-impl.jar:?]
     [exec] 	at com.liferay.portal.kernel.bean.BeanPropertiesUtil.getInteger(BeanPropertiesUtil.java:119) ~[portal-kernel.jar:?]
     [exec] 	at com.liferay.portal.kernel.bean.BeanParamUtil.getInteger(BeanParamUtil.java:246) ~[portal-kernel.jar:?]
     [exec] 	at com.liferay.portal.kernel.bean.BeanParamUtil.getInteger(BeanParamUtil.java:238) ~[portal-kernel.jar:?]


     [exec] 2022-07-15 18:59:05.815 ERROR [http-nio-8080-exec-7][BeanPropertiesImpl:297] null
     [exec] jodd.bean.BeanException: Simple property not found: suffixId. Invalid property: ContactImpl#suffixId (ContactImpl#suffixId)
     [exec] 	at jodd.bean.BeanUtilBean.getSimpleProperty(BeanUtilBean.java:206) ~[jodd-util.jar:6.0.1]
     [exec] 	at jodd.bean.BeanUtilBean._getIndexProperty(BeanUtilBean.java:307) ~[jodd-util.jar:6.0.1]

**Comment by gislayne-vitorino** [comment]
ci:test:sf

**Comment by gislayne-vitorino** [comment]
ci:test:relevant

**Comment by epark18** [comment]
ci:test:upgrade-all

**Comment by drewbrokke** [comment]
ci:test:relevant

**Comment by drewbrokke** [comment]
@gislayne-vitorino looks like there's a rebase error here.

**Comment by gislayne-vitorino** [comment]
ci:test:sf

**Comment by gislayne-vitorino** [comment]
ci:test:relevant

**Comment by gislayne-vitorino** [comment]
ci:test:sf

**Comment by gislayne-vitorino** [comment]
ci:test:relevant

**Comment by Sharryxue** [comment]
- Only semantic-versioning issue, refer [console log](https://testray.liferay.com/reports/production/logs/2022-07/test-1-27/test-portal-acceptance-pullrequest(master)/4167/modules-semantic-versioning-jdk8/0/0/jenkins-console.txt.gz?authuser=0) found, could you @gislayne-vitorino please take a look into the issue? Thanks in advance. No other related failure in ci:test:relevant.
```
     [exec] com.liferay.semantic.versioning.SemanticVersioningTest > testSemanticVersioning[/apps/portal-security/portal-security-ldap-api] FAILED
     [exec]     java.lang.AssertionError:   PACKAGE_NAME                                       DELTA      CUR_VER    BASE_VER   REC_VER    WARNINGS  
     [exec]     = ================================================== ========== ========== ========== ========== ==========
     [exec]     * com.liferay.portal.security.ldap.exportimport      MAJOR      1.1.0      1.1.0      2.0.0      VERSION INCREASE REQUIRED
     [exec]     	>   class      com.liferay.portal.security.ldap.exportimport.LDAPUser
     [exec]     		-   method     getPrefixId()
     [exec]     			-   return     long
     [exec]     		+   method     getPrefixListTypeId()
     [exec]     			+   return     long
     [exec]     		-   method     getSuffixId()
     [exec]     			-   return     long
     [exec]     		+   method     getSuffixListTypeId()
     [exec]     			+   return     long
     [exec]     [Baseline Warning] Bundle Version Change Recommended: 5.0.0
```

**Comment by Sharryxue** [comment]
ci:test:upgrade-all

**Comment by gislayne-vitorino** [comment]
ci:test:sf

**Comment by gislayne-vitorino** [comment]
ci:test:relevant

**Comment by epark18** [comment]
These test results look good! The only issue is the semantic versioning issue that @Sharryxue mentioned here https://github.com/liferay-usm/liferay-portal/pull/631#issuecomment-1200952003 is still appearing on these results.

**Comment by gislayne-vitorino** [comment]
ci:test:sf

**Comment by gislayne-vitorino** [comment]
ci:test:relevant

**Comment by epark18** [comment]
ci:test:relevant

**Comment by epark18** [comment]
@gislayne-vitorino Not sure why the failure didn't show up previously, but semantic versioning failed again in another module. 
```
     [exec] > Task :apps:users-admin:users-admin-test-util:baseline
     [exec]   PACKAGE_NAME                                       DELTA      CUR_VER    BASE_VER   REC_VER    WARNINGS  
     [exec] = ================================================== ========== ========== ========== ========== ==========
     [exec] * com.liferay.users.admin.test.util.search           MAJOR      4.0.0      4.0.0      5.0.0      VERSION INCREASE REQUIRED
     [exec] 	>   class      com.liferay.users.admin.test.util.search.UserBlueprintImpl
     [exec] 		-   method     getPrefixId()
     [exec] 			-   return     long
     [exec] 		+   method     getPrefixListTypeId()
     [exec] 			+   return     long
     [exec] 		-   method     getSuffixId()
     [exec] 			-   return     long
     [exec] 		+   method     getSuffixListTypeId()
     [exec] 			+   return     long
     [exec] 	>   interface  com.liferay.users.admin.test.util.search.UserBlueprint
     [exec] 		-   method     getPrefixId()
     [exec] 			-   access     abstract
     [exec] 			-   return     long
     [exec] 		+   method     getPrefixListTypeId()
     [exec] 			+   access     abstract
     [exec] 			+   return     long
     [exec] 		-   method     getSuffixId()
     [exec] 			-   access     abstract
     [exec] 			-   return     long
     [exec] 		+   method     getSuffixListTypeId()
     [exec] 			+   access     abstract
     [exec] 			+   return     long
     [exec] [Baseline Warning] Bundle Version Change Recommended: 6.0.0
```
I fetched this PR and ran baseline locally and added it here https://github.com/epark18/liferay-portal/pull/37/commits/8cb8c25b93c24b12e1607a89af18e44ba72ba67e. Going to test on my github https://github.com/epark18/liferay-portal/pull/37 once more and this one should be good to go as long as it passes there. Will update you with progress.

**Comment by gislayne-vitorino** [comment]
ci:test:upgrade

**Comment by epark18** [comment]
Forwarded to Brian https://github.com/brianchandotcom/liferay-portal/pull/121474.

**Comment by epark18** [comment]
ci:close

**Pull request closed without merging**